bundles/core/org.eclipse.smarthome.core/src/main/java/org/eclipse/smarthome/core/library/items/DimmerItem.java

```diff
@@ -96,8 +98,8 @@ public State getStateAs(Class<? extends State> typeClass) {
             return state.equals(PercentType.ZERO) ? OnOffType.OFF : OnOffType.ON;
         } else if (typeClass == DecimalType.class) {
             if (state instanceof PercentType) {
-                return new DecimalType(((PercentType) state).toBigDecimal().divide(new BigDecimal(100), 8,
-                        RoundingMode.UP));
+                return new DecimalType(

```

Comment:
This was a formatting change, but does highlight we were already converting between decimal and percent types in one direction.